molecule/core.py

```diff
@@ -234,6 +235,18 @@ def remove_inventory_file(self):
         if os._exists(self.config.config['ansible']['inventory_file']):
             os.remove(self.config.config['ansible']['inventory_file'])
 
+    def remove_vars_files(self):
+        group_vars_dir = os.path.join(
+            os.path.abspath(self.config.config['molecule']['molecule_dir']),
+            'group_vars')
+        host_vars_dir = os.path.join(
+            os.path.abspath(self.config.config['molecule']['molecule_dir']),
+            'host_vars')
+        if os.path.exists(group_vars_dir):
+            shutil.rmtree(group_vars_dir)
+        if os.path.exists(host_vars_dir):
+            shutil.rmtree(host_vars_dir)
+
```

Comment:
Tbh I am not extremely comfortable with pytest and all the tests in `test/unit/core/test_core.py` that would be testing something alike have a `@pytest.mark.skip(reason='TODO(retr0h): Determine best way to test this')` decorator, thus there is no sample to adapt from.